spec/controllers/v2/concerns/rendering_spec.rb

```diff
@@ -135,6 +135,22 @@
     expect(response.parsed_body).to match(expected)
   end
 
+  context 'referrable resource', if: [V2::ProfilesController, V2::RulesController].include?(described_class) do
```

Comment:
This logic is the opposite of declarative, instead please create a new file called `indexable_by_ref_id_spec.rb` under the concerns and define a shared example there that you explicitly specify.